src/day7.scala

```diff
@@ -0,0 +1,74 @@
+// using scala 3.1.0
+// using lib com.lihaoyi::pprint:0.6.6
+
+package day7
+
+import scala.util.Using
+import scala.io.Source
+import scala.annotation.tailrec
+
+case class Crabmarine(
+    horizontal: Int,
+    increasingCost: Boolean,
+    fuelCost: Int = 1
+):
+  def moveUp() =
```

Comment:
Shouldn't this be called `moveForward` and the other one `moveBackward`?

Reply:
Makes sense, since it's horizontal